test: update README.md file with the updated changes (#788)
osbuild/image-builder-crc: test/README.md

Image Builder needs a database running, the easiest way is to run a container like so:

```
sudo podman run -p 5432:5432 --name image-builder-db --health-cmd "pg_isready -u postgres -d imagebuilder" --health-interval 10s --health-timeout 5s --health-retries 5 -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=foobar -e POSTGRES_DB=imagebuilder postgres
```

Run `make build` and migrate the db (this only needs to be done once) to the latest version:

```
PGHOST=localhost PGPORT=5432 PGUSER=postgres PGPASSWORD=foobar PGDATABASE=imagebuilder MIGRATIONS_DIR=$HOME/image-builder/internal/db/migrations-tern ./image-builder-migrate-db-tern
```

> Open image-builder-migrate-db-tern.go file (under image-builder/cmd folder) and update 2 variables:

Reviewer: ```suggestion
Open `image-builder-migrate-db-tern.go` file (under `image-builder/cmd` folder) and update 2 variables:
```

> Open image-builder-migrate-db-tern.go file (under image-builder/cmd folder) and update 2 variables:

Reviewer: We could probably move this part before "Run `make build`...", what do you think?

Author: great idea

> 1) TernExecutable with path to 'tern executable location' 

Reviewer: ```suggestion
1) TernExecutable with path to tern executable location.
```

>    if you don't have tern tool install it -

Reviewer: ```suggestion
   If you don't have tern tool you can install it with -
```

>    go install github.com/jackc/tern/v2@latest

Reviewer: ```suggestion
   `go install github.com/jackc/tern/v2@latest`
```

2) TernMigrationsDir with path to migration dir -

>    "$HOME/image-builder/internal/db/migrations-tern"

Reviewer: ```suggestion
   `$HOME/image-builder/internal/db/migrations-tern`
```

OSBUILD_CERT_PATH="/etc/osbuild-composer/client-crt.pem" OSBUILD_KEY_PATH="/etc/osbuild-composer/client-key.pem"
```

> Otherwise set some other values and Start Image Builder:

Reviewer: ```suggestion
Otherwise set some other values and start Image Builder:
```
